### DISCUSSION TITLE
Error simulating multiple endpoints

### DISCUSSION THREAD

**Discussion by soutomas** [nlmixr2/rxode2#597] [Q&A]
See this error when simulating a model with 2 endpoints: 
![image](https://github.com/nlmixr2/rxode2/assets/32617414/7840f728-b714-411d-9530-2e03db1b06ca)

It was okay with either of the endpoint alone, e.g., if I comment out the line with the other endpoint as below:
![image](https://github.com/nlmixr2/rxode2/assets/32617414/0a28468f-5efd-4497-9480-313ae836c274)

Not sure why?

**Comment by mattfidler** [comment]
I need more information to answer exactly why.  However,  the [nlmixr2 description of multiple endpoints](https://nlmixr2.org/articles/multiple-endpoints.html) may be helpful and could possibly answer your question if you are not sure you can share the model or data.

**Comment by soutomas** [comment]
Thanks @mattfidler,
To test this out, I used the example model shown in [nlmixr2 description of multiple endpoints](https://nlmixr2.org/articles/multiple-endpoints.html) as suggested, but have the same error: 
```
pk.turnover.emax <- function() {
  ini({
    tktr <- log(1)
    tka <- log(1)
    tcl <- log(0.1)
    tv <- log(10)
    ##
    eta.ktr ~ 1
    eta.ka ~ 1
    eta.cl ~ 2
    eta.v ~ 1
    prop.err <- 0.1
    pkadd.err <- 0.1
    ##
    temax <- logit(0.8)
    #temax <- 7.5
    tec50 <- log(0.5)
    tkout <- log(0.05)
    te0 <- log(100)
    ##
    eta.emax ~ .5
    eta.ec50  ~ .5
    eta.kout ~ .5
    eta.e0 ~ .5
    ##
    pdadd.err <- 10
  })
  model({
    ktr <- exp(tktr + eta.ktr)
    ka <- exp(tka + eta.ka)
    cl <- exp(tcl + eta.cl)
    v <- exp(tv + eta.v)
    ##
    #poplogit = log(temax/(1-temax))
    emax=expit(temax+eta.emax)
    #logit=temax+eta.emax
    ec50 =  exp(tec50 + eta.ec50)
    kout = exp(tkout + eta.kout)
    e0 = exp(te0 + eta.e0)
    ##
    DCP = center/v
    PD=1-emax*DCP/(ec50+DCP)
    ##
    effect(0) = e0
    kin = e0*kout
    ##
    d/dt(depot) = -ktr * depot
    d/dt(gut) =  ktr * depot -ka * gut
    d/dt(center) =  ka * gut - cl / v * center
    d/dt(effect) = kin*PD -kout*effect
    ##
    cp = center / v
    cp ~ prop(prop.err) + add(pkadd.err)
    effect ~ add(pdadd.err)
  })
}
m1 = nlmixr2(pk.turnover.emax)
ev = et(amt=150000, until=48, ii=24, cmt=1) 
sim = m1 %>% rxSolve(events=ev,nSub=1) 
```

![image](https://github.com/nlmixr2/rxode2/assets/32617414/ecca463c-380a-4bb1-ab30-95033280a3b3)
